lib/Runtime/Language/ValueType.h

```diff
@@ -150,6 +150,7 @@ class ValueType
 
     bool HasBeenString() const;
     bool IsString() const;
+    bool HasHadStringTag() const; // Note: this is likely not what you're looking for; this is a short-cut for LdLen optimization.
```

Comment:
I've expanded the comment now and moved it so that intellisense can properly pick it up.